Userland/Applications/DisplaySettings/BackgroundSettingsWidget.cpp

```diff
@@ -7,19 +7,20 @@
  */
 
 #include "BackgroundSettingsWidget.h"
+#include <AK/LexicalPath.h>
```

Comment:
Is it really needed? I guess that it was indirectly included by `FilePicker.h`.

Reply:
Ya it is. It's an indirect include exactly as you said.